src/util.js

```diff
@@ -21,9 +21,107 @@ const formatList = (list) => {
   return listPlugin
 }
 
-const ucfirst = (str) => `${str.charAt(0).toUpperCase()}${str.slice(1)}`
+const groupByPattern = (list, pattern) => {
+  const mixins = []
+  list.forEach(str => {
+    if (str.indexOf(pattern) !== -1) {
+      mixins.push(str)
+    }
+  })
 
-const uniqArray = (array) => array.filter((elem, pos, arr) => arr.indexOf(elem) === pos)
+  return mixins
+}
+
+const formatScssList = list => {
+  let result = []
+  const mixinList = groupByPattern(list, 'mixins')
+  const utilitiesList = groupByPattern(list, 'utilities')
+
+  const functionsIndex = list.findIndex(file => file.indexOf('functions') !== -1)
+  if (functionsIndex !== -1) {
+    result.push(list[functionsIndex])
+  }
+
+  const varIndex = list.findIndex(file => file.indexOf('variables') !== -1)
+  if (varIndex !== -1) {
+    result.push(list[varIndex])
+  }
+
+  // Add mixins
+  result = result.concat(mixinList)
+  // Remove mixins from the list
+  mixinList.forEach(mixin => {
+    const mixinIndex = list.indexOf(mixin)
+    list.splice(mixinIndex, 1)
+  })
+
+  const plugins = [
+    'root',
+    'reboot',
+    'type',
+    'images',
+    'code',
+    'gird',
+    'tables',
+    'forms',
+    'buttons',
+    'transitions',
+    'dropdown',
+    'button-group',
+    'input-group',
+    'custom-form',
```

Comment:
Shouldn't this be _custom-form**s**_?

Reply:
Yep you're right but it works like that too, I'll change that 👍

Reply:
I haven't tested this, I was just comparing the list order with the Bootstraps order and noticed this 🙂